Mathlib/Algebra/MvPolynomial/Equiv.lean

```diff
@@ -454,8 +453,14 @@ theorem finSuccEquiv_support (f : MvPolynomial (Fin (n + 1)) R) :
     refine ⟨tail m, ?_⟩
     rwa [← coeff, zero_apply, ← mem_support_iff, support_coeff_finSuccEquiv, cons_tail]
 
-theorem finSuccEquiv_support' {f : MvPolynomial (Fin (n + 1)) R} {i : ℕ} :
-    Finset.image (Finsupp.cons i) (Polynomial.coeff ((finSuccEquiv R n) f) i).support =
+@[deprecated (since := "2024-11-05")] alias finSuccEquiv_support := support_finSuccEquiv
+
+theorem mem_support_finSuccEquiv {f : MvPolynomial (Fin (n + 1)) R} {x} :
+    x ∈ (finSuccEquiv R n f).support ↔ x ∈ (fun m : Fin (n + 1) →₀ _ ↦ m 0) '' f.support := by
+  simpa using congr(x ∈ $support_finSuccEquiv)
+
+theorem image_support_finSuccEquiv {f : MvPolynomial (Fin (n + 1)) R} {i : ℕ} :
+    Finset.image (Finsupp.cons i) (Polynomial.coeff (finSuccEquiv R n f) i).support =
```

Comment:
```suggestion
    (Polynomial.coeff (finSuccEquiv R n f) i).support.image (Finsupp.cons i) =
```
If this works (untested) it looks easier to read, and makes the name change look more reasonable

Reply:
(plus dot notation for Polynomial.coeff, if that works too)